trieste/models/gpflux/__init__.py

```diff
@@ -21,3 +21,10 @@
 from .builders import build_vanilla_deep_gp
 from .interface import GPfluxPredictor
 from .models import DeepGaussianProcess
+from .sampler import (
```

Comment:
Are all of these really worth exporting from trieste.models.gpflux or are some of them implementation details that can remain in trieste.models.gpflux.sampler?

Reply:
I have just noticed that nothing was exported and took a look at gpflow, there everything was exported so I have jusst copied the approach